canonical/ubuntu.com
# Build activate page to redeem activation key for OEM store

## Done

- Build activate page to redeem activation key for OEM store
- UX & Design will be updated when its ready. 
- Use any of these activation keys [here](https://pastebin.canonical.com/p/NsPJbhKs6H/)

## QA

- [x] Check activate page is accessible only for UO users.

1. Go to /pro/activate page without log in
2. Check activate page is not accessible 

- [x] Get paid account for new users (Since a new user's account without any purchase history is not a paid account, we need to make the account a paid account by submitting proper account name) 
1. Create new UO account
2. Go to  /pro/activate
4. Log in on ubnutu.com
5. See "buying-for" form is showing 
6. Check form submission working properly when buying for myself and organisation. Organisation name is required when organisation is selected.
7. Check it shows /pro/activate page after submitting the form.

- [x] Activate key (Only paid account can activate keys) 
1. Redeem activate key and QA with wrong keys too.
2. Check the notification message shows when its failed
3. Check page redirects to /pro/dashboard

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-3984
